src/main/webapp/app/shared/markdown-editor/commands/command.ts

```diff
@@ -1,6 +1,21 @@
 import { ElementRef } from '@angular/core';
 import { IconProp } from '@fortawesome/fontawesome-svg-core';
 
+// Work around to update the range
+const ace = require('brace');
```

Comment:
Can we use  ``import`` instead? We're not using ``require`` anywhere else in our application code

Reply:
I guess that should also work :D